smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/ImportDeclarations.java

```diff
@@ -63,34 +69,41 @@ ImportDeclarations addImport(String name, String alias, String module) {
     public String toString() {
         StringBuilder result = new StringBuilder();
 
-        if (!imports.isEmpty()) {
-            for (Map.Entry<String, Map<String, String>> entry : imports.entrySet()) {
+        if (!defaultImports.isEmpty()) {
+            List<Map.Entry<String, String>> entries = new ArrayList<>(defaultImports.entrySet());
+            for (Map.Entry<String, String> importEntry : entries) {
+                result.append("import ")
+                        .append(importEntry.getValue())
+                        .append(" from \"")
+                        .append(importEntry.getKey())
+                        .append("\";\n");
+            }
+            result.append("\n");
+        }
+
+        if (!namedImports.isEmpty()) {
+            for (Map.Entry<String, Map<String, String>> entry : namedImports.entrySet()) {
                 String module = entry.getKey();
                 Map<String, String> moduleImports = entry.getValue();
-                List<Map.Entry<String, String>> nonStarEntries = new ArrayList<>(moduleImports.entrySet());
+                List<Map.Entry<String, String>> entries = new ArrayList<>(moduleImports.entrySet());
```

Comment:
Instead of converting the Set to a List, just do:

```
Set<Map.Entry<String, String>> entries = moduleImports.entrySet();
```

Reply:
This PR just renames the List instead of introducing new type, but fixed in [`3f34b04` (#212)](https://github.com/awslabs/smithy-typescript/pull/212/commits/3f34b04375fc742c1e22e44ab953f43f81bb559a)